WordPress/src/main/java/org/wordpress/android/ui/jetpack/backup/download/handlers/BackupDownloadHandler.kt

```diff
@@ -0,0 +1,55 @@
+package org.wordpress.android.ui.jetpack.backup.download.handlers
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MediatorLiveData
+import kotlinx.coroutines.CoroutineDispatcher
+import kotlinx.coroutines.flow.collect
+import kotlinx.coroutines.flow.flowOn
+import org.wordpress.android.fluxc.model.SiteModel
+import org.wordpress.android.fluxc.store.ActivityLogStore.BackupDownloadRequestTypes
+import org.wordpress.android.modules.BG_THREAD
+import org.wordpress.android.ui.jetpack.backup.download.handlers.BackupDownloadHandler.BackupDownloadHandlerStatus.Failure
+import org.wordpress.android.ui.jetpack.backup.download.handlers.BackupDownloadHandler.BackupDownloadHandlerStatus.Success
+import org.wordpress.android.ui.jetpack.backup.download.usecases.PostBackupDownloadUseCase
+import org.wordpress.android.ui.pages.SnackbarMessageHolder
+import org.wordpress.android.ui.utils.UiString
+import org.wordpress.android.viewmodel.Event
+import javax.inject.Inject
+import javax.inject.Named
+
+class BackupDownloadHandler @Inject constructor(
```

Comment:
❓ Do we plan to add more logic into this class in the future? It feels like it doesn't do anything and it adds a lot of complexity (eg. we need to observerForever in the VM). Wdyt?

Reply:
@malinajirka - Guess I should have read all your comments before posting my first response. Yes, I can eliminate the handlers all together and move this logic directly to the view models. :)